plugins/plugin-chart-concise-card/src/plugin/utils.tsx

```diff
@@ -0,0 +1,11 @@
+export function transformFilters(
+  filters: { subject: string; operator: string; comparator: string }[],
+) {
+  return filters.map(filter => {
+    return {
+      col: filter.subject,
+      op: filter.operator,
+      val: filter.comparator,
+    };
+  });
```

Comment:
this looks really familiar and i think it exists somewhere else already, but when i went looking i couldn't find it.  just a note.